packages/babel-core/src/transformation/normalize-file.ts

```diff
@@ -36,7 +36,7 @@ export default function* normalizeFile(
     }
 
     if (options.cloneInputAst) {
-      ast = cloneDeep(ast);
+      ast = cloneDeep(ast) as t.File;
```

Comment:
you could fix type of cloneDeep instead to something like this

https://github.com/babel/babel/blob/52cdab724d84efaea6a0cf5b4d1d993df6576257/packages/babel-core/src/transformation/util/clone-deep.ts#L4

```ts
export default function <T> (value: T): T {
```

Reply:
Yes, because there is already an existing PR that completely replaced cloneDeep, so I did not modify it, thank you!